docs/plugins/vault.md

```diff
@@ -0,0 +1,42 @@
+# *vault* Plugin
+
+This plugin installs [vault](https://www.vaultproject.io/) in an existing Kubernetes cluster using the Helm chart.
+
+### Usage
+
+```yaml
+tools:
+- name: vault
+  # name of the plugin
+  plugin: vault
+  options:
+    # need to create the namespace or not, default: false
+    create_namespace: true
+    repo:
+      # name of the Helm repo
+      name: hashicorp
+      # url of the Helm repo
+      url: https://helm.releases.hashicorp.com
+    # Helm chart information
+    chart:
+      # name of the chart
+      chart_name: hashicorp/vault
+      # release name of the chart
+      release_name: vault
+      # k8s namespace where Vault will be installed
+      namespace: devstreamtest1
```

Comment:
It would be better to use a meaningful name. e.g. vault/hashicorp/... Maybe the docs with vault has some example.

Reply:
I agree, in the documentation, please put a more proper example value for "namespace."